build-system/default-pre-push

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+#
+# Copyright 2018 The AMP HTML Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the license.
+#
+# This script adds a pre-push hook to .git/hooks/, which runs some basic tests
+# before running "git push".
+#
+# Default pre-push hook for AMPHTML. To enable, run:
+#     "./build-system/enable-git-pre-push.sh"
```

Comment:
Out of curiosity, what the plan for Windows developers?

Reply:
I believe Windows users install `git` in order to develop AMP, and therefore, should already be able to run scripts such as this one. I'd be willing to try this if you have a Windows machine. In any case, since this isn't a required part of the development flow, we can always update it at a later time.

Reply:
Perhaps leave a comment or something similar about Windows development.

Reply:
Will do this when I add it to the Dev docs. Shouldn't be long before we know if this works or not!